package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "javascriptprojects",
+  "version": "1.0.0",
+  "description": "html,css,js practice\r # Web design",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Can you set up Jest configuration in the test property?

Reply:
Sure, I'll do

Reply:
Sure, I'll do it